src/com/epam/javacourse/homework4/HousePlant.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.javacourse.homework4;
+
+public class HousePlant extends Plant {
+
+    private final boolean needsSunlight;
+
+    public HousePlant(String name, String type, String nativeRegion, boolean needsSunlight) {
```

Comment:
Boolean variables naming can start with `is` and `has`. I suggest renaming it as `hasNeedForSunlight` or in another more natural way.

Reply:
Done